include/mbedtls/psa_util.h

```diff
@@ -509,6 +509,6 @@ extern mbedtls_psa_drbg_context_t *const mbedtls_psa_random_state;
 
 #endif /* !defined(MBEDTLS_PSA_CRYPTO_EXTERNAL_RNG) */
 
-#endif /* MBEDTLS_PSA_CRYPTO_C */
+#endif /* defined(MBEDTLS_USE_PSA_CRYPTO) || defined(MBEDTLS_SSL_PROTO_TLS1_3) */
```

Comment:
```suggestion
#endif /* MBEDTLS_USE_PSA_CRYPTO || MBEDTLS_SSL_PROTO_TLS1_3 */
```

Reply:
I used this format to be consistent with other similar comments in this file:

https://github.com/ARMmbed/mbedtls/blob/3f076dfb6dafde4048a90b1045c64591d34e55f7/include/mbedtls/psa_util.h#L495-L512

Reply:
ok, thanks